sim/druid/item_sets_pve_phase_6.go

```diff
@@ -183,15 +183,33 @@ var ItemSetGenesisFury = core.NewItemSet(core.ItemSet{
 
 // Each time you Dodge while in Dire Bear Form, you gain 10% increased damage on your next Mangle or Swipe, stacking up to 5 times.
 func (druid *Druid) applyTAQGuardian2PBonus() {
-	label := "S03 - Item - TAQ - Druid - Guardian 2P Bonus"
-	if druid.HasAura(label) {
-		return
-	}
+	druid.Tank2PieceAqProcAura = druid.RegisterAura(core.Aura{
+		Label:     "Guardian 2P Bonus Proc",
+		ActionID:  core.ActionID{SpellID: 1213188},
+		Duration:  time.Second * 10,
+		MaxStacks: 5,
+		OnStacksChange: func(aura *core.Aura, sim *core.Simulation, oldStacks, newStacks int32) {
+			druid.MangleBear.DamageMultiplierAdditive += 0.1 * float64(newStacks-oldStacks)
```

Comment:
This would need a guard checking `if druid.MangleBear != nil` to prevent erroring if not using the Mangle rune